check_license.sh

```diff
@@ -21,17 +21,18 @@ license_header="// Copyright ${year} ${license_copyright}
 
 function buildExcludeDirectories() {
   local result
```

Comment:
Is this line still needed if you have the other `local result=...` line?

Reply:
oups, indeed not needed anymore.